projects/aca-folder-rules/src/lib/rule-details/conditions/rule-simple-condition.ui-component.html

```diff
@@ -22,6 +22,14 @@
   </mat-form-field>
 
   <mat-form-field class="aca-rule-simple-condition__form__parameter-input">
-    <input matInput placeholder="{{ 'ACA_FOLDER_RULES.RULE_DETAILS.PLACEHOLDER.VALUE' | translate }}" type="text" formControlName="parameter" data-automation-id="value-input">
+    <mat-select formControlName="parameter" data-automation-id="value-select" *ngIf="selectedField?.name === 'mimetype'; else valueInput">
+      <mat-option *ngFor="let mimeType of mimeTypes"
+        [value]="mimeType.value">
+        {{ mimeType.label }}
+      </mat-option>
+    </mat-select>
+    <ng-template #valueInput>
+      <input matInput placeholder="{{ 'ACA_FOLDER_RULES.RULE_DETAILS.PLACEHOLDER.VALUE' | translate }}" type="text" formControlName="parameter" data-automation-id="value-input">
```

Comment:
can you check if automation id is used anywhere, and remove from this element if not

Reply:
removed automation-id